health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/RenderSelectionField.js

```
@@ -0,0 +1,48 @@
+const RenderSelectionField = ({ field, t }) => {
+  const isMdmsEnabled = field?.isMdms && !!field?.schemaCode;
+
+  const { isLoading, data } = Digit?.Hooks.useCustomMDMSuseCustomMDMS(
+    Digit?.ULBService?.getStateId(),
+    field?.schemaCode?.split(".")[0],
+    [{ name: field?.schemaCode?.split(".")[1] }],
+    {
+      select: (data) => {
+        const optionsData = _.get(data, `${field?.schemaCode?.split(".")[0]}.${field?.schemaCode?.split(".")[1]}`, []);
+        return optionsData
+          .filter((opt) => (opt?.hasOwnProperty("active") ? opt.active : true))
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)